.github/workflows/ios_build_and_test.yml

```
@@ -2,22 +2,18 @@ name: ios_build_and_test
 
 on:
   push:
-    branches: [ main ]
-  pull_request:
-    branches: [ main ]
 
 jobs:
   build:
     strategy:
       matrix:
         macos: [macos-latest]
```

Looks like `macos-latest` still points to macOS 12.
```suggestion
        macos: ['macos-13']
```